decorator.rb

```diff
@@ -0,0 +1,24 @@
+require './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
```

Comment:
- Please, kindly add the attr_accessor for the nameable which creates both the reader & writer methods.

Reply:
Alright, thanks @zieeco  I will do well to make implementations immediately.